en/components/overlay_main.md

```diff
@@ -40,11 +34,11 @@ export class MyOverlayComponent {
 
 The overlay service can be used to dynamically display a `HTMLNode` or even an Angular Component by attaching it to the overlay DOM.
 
-After a reference to the Overlay service is established, it can be used to dynamically show/hide content. For example, we can pass an Angular Component in the `attach` function. This will generate unique ID. We can pass then this ID to `show` function to display the component:
+After a reference to the Overlay service is established, it can be used to dynamically show/hide content. For example, we can pass an Angular Component in the `attach` function. This will generate a unique ID, which we can pass to the `show` function to display the component:
```

Comment:
On all other places bellow in the document `attach` and `show` are described as methods. Only here we say `function`. Can you please change this.